src/components/NcListItem/NcListItem.vue

```diff
@@ -233,12 +262,18 @@
 								<slot name="subtitle" />
 							</span>
 
-							<!-- Counter -->
-							<span v-if="showCounter" class="line-two__counter">
+							<!-- Counter and indicator -->
+							<span v-if="showAdditionalElements" class="line-two__additional_elements">
 								<NcCounterBubble v-if="counterNumber != 0"
+									class="line-two__counter"
 									:type="counterType">
 									{{ counterNumber }}
 								</NcCounterBubble>
+
+								<span class="line-two__indicator">
```

Comment:
Since this element has a margin, it will take space even if the slot is empty, which moves the CounterBubble to the left then.
```suggestion
								<span v-if="hasIndicator" class="line-two__indicator">
```
and add a `hasIndicator` variable, similar to e.g. `hasActions`.

Reply:
please write why should i use hasIndicator (ok, besides additional span) and how should i define it, i  don't know right now

Reply:
I just named it accordingly to the `hasActions` and `hasSubtitle` variables. It is just meant to hide the `line-two__indicator` entirely if the slot is empty.
Just put
```js
 			if (this.hasIndicator !== !!this.$slots.indicator) { 
 				this.hasIndicator = !!this.$slots.indicator
 			}
```
here
https://github.com/nextcloud/nextcloud-vue/blob/01024eb24f66c0fd015c088e7695ed2567c986c9/src/components/NcListItem/NcListItem.vue#L582-L588

and 
```js
			hasIndicator: false,
```
here
https://github.com/nextcloud/nextcloud-vue/blob/01024eb24f66c0fd015c088e7695ed2567c986c9/src/components/NcListItem/NcListItem.vue#L453

I think this should do it.

Reply:
ok, done
Thank you!